### PR TITLE
Adding support for windows-arm64

### PR DESCRIPTION
**Pull request by koushdey**
This PR adds changes to support ORAS CLI on windows 64 Architecture.

### CODE REVIEW

**Review by phoenix24 — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -64,10 +64,17 @@ build-mac-arm64:
 		-o bin/darwin/arm64/$(CLI_EXE) $(CLI_PKG)
 
 .PHONY: build-windows
-build-windows:
+build-windows: build-windows-amd64 build-windows-arm64
+
+build-windows-amd64:
```
Comment (qweeah) [inline Makefile:69]:
```suggestion
.PHONY: build-windows-amd64
build-windows-amd64:
```

**Comment by qweeah** [comment]
@koushdey Can you follow the  [instructions ](https://github.com/oras-project/oras/pull/455/checks?check_run_id=7448504152) to resolve DCO check failure? Thanks.

**Comment by qweeah** [comment]
@shizhMSFT @FeynmanZhou If this is merged, we also need to update https://github.com/oras-project/oras-www/blob/main/docs/CLI/5_developer_guide.md and add windows arm64 binaries in.

**Comment by koushdey** [comment]
Unable to push the signed off changes. I am closing this PR and opening a fresh one.

**Pull request closed without merging**